### PR TITLE
Renovation: remove testing/jest folder, test utils refactoring

### PR DESCRIPTION
**Pull request by churkin**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -25,19 +25,15 @@ module.exports = {
             branches: 0
```
Comment (MaximKudriavtsev) [inline jest.config.js:25]: When we update these values?

**Review by MaximKudriavtsev — DISMISSED** [review]

File: jest.config.js (hunk 1 above)
Comment (churkin) [reply jest.config.js:25]: Do all tests already 100% cover renovation code?

Comment (MaximKudriavtsev) [reply jest.config.js:25]: I think no. But we can increase these properties to some values to get rid of adding widgets without testing. And we can ask some guys to increase code coverage rating.

Comment (churkin) [reply jest.config.js:25]: @ksercs will do it

**Review by roman-simionov — DISMISSED** [review]

**Review by MaximKudriavtsev — DISMISSED** [review]

**Review by roman-simionov — APPROVED** [review]

**Pull request merged**